--- a/platform/generic/objects.mk	2024-12-24 09:58:18.000000000 +0000
+++ b/platform/generic/objects.mk	2025-05-11 11:35:35.673520926 +0100
@@ -31,7 +31,7 @@
   # This needs to be 2MB aligned for 64-bit system
   FW_JUMP_OFFSET=0x200000
 endif
-FW_JUMP_FDT_OFFSET=0x2200000
+FW_JUMP_FDT_OFFSET=0x30000000
 FW_PAYLOAD=y
 ifeq ($(PLATFORM_RISCV_XLEN), 32)
   # This needs to be 4MB aligned for 32-bit system
